core/src/main/scala/org/apache/spark/scheduler/TaskScheduler.scala

```diff
@@ -109,4 +109,7 @@ private[spark] trait TaskScheduler {
    */
   def applicationAttemptId(): Option[String]
 
+  def markPartitionIdAsCompletedAndKillCorrespondingTaskAttempts(
```

Comment:
We already have `submitTasks`, `cancelTasks`, `killTaskAttempt` etc. in this class, how about  renaming this method to `completeTasks`? And add comment to explain what this method does.

Reply:
Done